src/esp/sensor/CameraSensor.cpp

```diff
@@ -55,7 +57,7 @@ void CameraSensor::recomputeBaseProjectionMatrix() {
   // refresh size after relevant parameters have changed
   Mn::Vector2 nearPlaneSize_ =
       Mn::Vector2{1.0f, static_cast<float>(height_) / width_};
-  float scale;
+  float scale = NAN;
```

Comment:
NAN means not-a-number.
And scale will be used later in `nearPlaneSize_ /= scale;`
I think `scale = 1.0f` would be a reasonable initial value.

Reply:
I would prefer NaN, as scale SHOULD be set. The reason to initially set it as NaN is to know something has gone horribly wrong in the constructor or elsewhere.

Scale will only be user later in nearPlaneSize_ /= scale after being parsed from the config value.